cmd/radikron-gui/README.md

```
@@ -0,0 +1,119 @@
+Build the application:
+```bash
+wails build
+```
+
+This creates platform-specific binaries in `build/bin/`.
+
+## Architecture
+
+### Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)